core/server/worker/src/main/java/alluxio/worker/grpc/GrpcDataServer.java

```diff
@@ -63,6 +64,8 @@
   private EventLoopGroup mBossGroup;
   private EventLoopGroup mWorkerGroup;
   private GrpcServer mServer;
+  private final FileSystemContext mFsContext =
```

Comment:
Usually we group by field modifier, i.e. all the `private static final` in one group, then `private final`, then `private`

Reply:
fixed